yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -348,7 +346,7 @@ export class AztecNodeService implements AztecNode {
       await getTreeRoot(MerkleTreeId.BLOCKS_TREE),
       Fr.ZERO,
       await getTreeRoot(MerkleTreeId.PUBLIC_DATA_TREE),
-      globalsHash,
+      await this.merkleTreeDB.getLatestGlobalVariablesHash(),
```

Comment:
hmm is there any chance this would advance such that the global variable hash is too new? Like if the system moves on after `await getTreeRoot(MerkleTreeId.PUBLIC_DATA_TREE),`

Reply:
I don't fully understand the code, but if we're fetching data it would be good to fetch it all at once representing one snapshot, 'latest' makes me nervous when used with other async calls

Reply:
These changes have unfortunately broken the p2p tests, this could be the culprit